client-rpc/src/rpc/websocket_rpc.rs

```diff
@@ -0,0 +1,154 @@
+use crate::rpc::websocket_core::WebsocketCore;
+use chain_core::state::account::StakedStateAddress;
+use client_common::tendermint::Client;
+use client_common::{PrivateKey, PublicKey, Storage};
+use client_core::WalletClient;
+use client_index::BlockHandler;
+use futures::future::Future;
+use futures::sink::Sink;
+use futures::stream::Stream;
+use futures::sync::mpsc;
+use std::thread;
+use websocket::result::WebSocketError;
+use websocket::{ClientBuilder, OwnedMessage};
+
+// this handles low level network connection
```

Comment:
The convention in Rust is that `/// comment` is used for top-level (on consts, structs, functions...) comments that would be picked up by `cargo docs` if you generate the HTML documentation.
`// comment` is used for inline comments inside code blocks

Reply:
ok